plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/serialize/SerializerDescriptor.java

```diff
@@ -33,22 +34,26 @@ public class SerializerDescriptor extends AbstractDescriptor
 
     private String id;
     private ObjectType serializerType;
+    private boolean distributed;
 
-    SerializerDescriptor(IConfigurationElement config)
-    {
+    SerializerDescriptor(IConfigurationElement config) {
         super(config);
 
         this.id = config.getAttribute("id");
         this.serializerType = new ObjectType(config.getAttribute("class"));
+        String distributedAttr = config.getAttribute("distributed");
+        this.distributed = CommonUtils.isEmpty(distributedAttr) || CommonUtils.toBoolean(distributedAttr);
```

Comment:
Is it expected to be `true` when the `distributed` attribute is missing?

Anyways, you can use `CommonUtils#getBoolean` and pass the default value directly.

```suggestion
        this.distributed = CommonUtils.getBoolean(config.getAttribute("distributed"), true);
```